### ISSUE TITLE
Moderate security vulnerability for versions of Axios 0.8.1 - 1.5.1 (Dependency)

### ISSUE THREAD

**Issue by OIRNOIR** [keymetrics/pm2-io-js-api#76]
This shows a moderate security vulnerability in the `npm audit` text, since PM2 depends on this package. The fixed version is Axios 1.6.0 and higher. I suggest we migrate towards that one.

**Comment by RobinTail** [comment]
Reference: https://github.com/advisories/GHSA-wf5p-g6vw-rhxx

**Comment by globalexport** [comment]
Maybe it is time to leave axios for another library since the used v0.21.4 is far away from current state. Undici perhaps?

**Comment by buffcode** [comment]
@globalexport Axios is maintained. `pm2/js-api` isn't maintained enough to keep up with dependency updates ;)

**Comment by globalexport** [comment]
@buffcode You are right. Although it would be nice to avoid having such a dependency at all. I am getting audit failures for dozens of projects just because of this case everyday. And it's open for two weeks now. :/

**Comment by MartinFalatic** [comment]
Any package that isn't maintained enough to keep up with vulnerabilities is effectively dead. Either the vulnerability should be fixed or the package replaced.

**Comment by OIRNOIR** [comment]
> Any package that isn't maintained enough to keep up with vulnerabilities is effectively dead. Either the vulnerability should be fixed or the package replaced.

The package itself's vulnerability has been fixed. We are just using a severely outdated version of said package. We need to update the package or else it is *this* project that becomes "effectively dead".

**Comment by JackHammer29** [comment]
Any update on this please?

**Comment by OIRNOIR** [comment]
> Any update on this please?

We're still waiting for @Unitech to review my PR. It's honestly getting annoying seeing Dependabot email me about this vulnerability.

**Comment by guizzo** [comment]
@Unitech Any update?

**Comment by estevanpedro** [comment]
How do i fix this? Should i stop using pm2? What are the others options/solutions?

**Comment by smo043** [comment]
Added details here - https://github.com/Unitech/pm2/issues/5728

**Comment by boxexchanger** [comment]
@Unitech any update?

**Comment by Unitech** [comment]
fixed via pm2@5.3.1

**Comment by kopacko** [comment]
The UUID version issue is not fixed with pm2 v5.3.1.

```
/usr/lib
├─┬ pm2@5.3.1
│ └─┬ @pm2/io@5.0.2
│   ├─┬ @opencensus/core@0.0.9
│   │ └── uuid@3.4.0
│   └─┬ @opencensus/propagation-b3@0.0.8
│     ├─┬ @opencensus/core@0.0.8
│     │ └── uuid@3.4.0 deduped
│     └── uuid@3.4.0 deduped
└── uuid@9.0.1
```
